nilearn/plotting/html_surface.py

```diff
@@ -118,8 +118,8 @@ def _one_mesh_info(
         vmax=vmax,
         vmin=vmin,
     )
-    info = {"inflated_left": mesh_to_plotly(surf_mesh)}
-    info["vertexcolor_left"] = get_vertexcolor(
+    info = {"inflated_both": mesh_to_plotly(surf_mesh)}
+    info["vertexcolor_both"] = get_vertexcolor(
```

Comment:
> The plotly engine doesn't work anymore (issue with `view_surf`).

This fixes `view_surf`. Seems like I introduced it in #4991. But that shouldn't have affected the stable docs. Weird...